### PR TITLE
The selector for the allow button use the resourceId instead of the index

### PR DESCRIPTION
**Pull request by crgarridos**
See #108 

Is the fist time I submit a pull request, hope I'm not messing up the things :s

### CODE REVIEW

**Comment by rocboronat** [comment]
Nice, thanks @crgarridos ! Let me pass the tests on some devices I have around... :·D

**Comment by rocboronat** [comment]
Nice, it works! Going to test it on emulators with all versions with runtime permissions: Android 23, 24, 25, 26... just to check that it works great :·)

**Comment by rocboronat** [comment]
Let me share the results of the tests...

Android 23: OK!
Android 24: OK!
Android 25: OK!
Android 26: OK!

😄 Thanks for the work, @crgarridos !

File: library/src/main/java/com/schibsted/spain/barista/permission/PermissionGranter.java [hunk 1]
```diff
@@ -17,7 +17,8 @@
 public class PermissionGranter {
 
   private static final int PERMISSIONS_DIALOG_DELAY = 3000;
-  private static final int GRANT_BUTTON_INDEX = 1;
+  private static final String PERMISSIONS_DIALOG_ALLOW_ID = "com.android.packageinstaller:id/permission_allow_button";
+//    private static final String PERMISSIONS_DIALOG_DENY_ID = "com.android.packageinstaller:id/permission_deny_button";
```
Comment (rocboronat) [inline library/src/main/java/com/schibsted/spain/barista/permission/PermissionGranter.java]: We will create another method that will use this ID to deny a permission. Thanks for it!

**Comment by crgarridos** [comment]
Great ! Thank to you for the awesome library :)

**Pull request merged**